From b24af2914c473f7055b76a56a9b13a24e397dd34 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Tue, 27 Oct 2026 12:35:02 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh arunisaac.pub from keyservers/WKD: same certificate(s), with
self-signatures our copy was missing; the commands are in the spec
comment.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 arunisaac.pub | 257 +++++++++++++++++++++++++++++++++++++-------------
 skribilo.spec |  12 ++-
 2 files changed, 199 insertions(+), 70 deletions(-)

diff --git a/arunisaac.pub b/arunisaac.pub
index f793ba6..cadf966 100644
--- a/arunisaac.pub
+++ b/arunisaac.pub
@@ -1,75 +1,196 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQENBFIrWlgBCADL9VFDowZ3tM5Ogb38eC3I9PWhXpEVV6DB7AKJlOdpdc5caBng
+xsBNBFIrWlgBCADL9VFDowZ3tM5Ogb38eC3I9PWhXpEVV6DB7AKJlOdpdc5caBng
 CUu6UfnVwtNXT3eTr+vG5u3tANnaVbL6tq+8o7V1gWhpAlQ62OfgolxExpHOjHII
 xUrPyi4JdNnVyYp/GfEb7hHz4zN0nY6CyVn8DFXOCmJzQvnAWFHEKjtCLFWguxO9
 e/nElbBUo+CNa7eUurKW/cCym0/bkLZBMTXlwoy6ZAik5w92gNUyQ7x8ICKTzmPF
 HQVZ+gN7fNWh2KjP3DFU1KV909xKnaQz2FZcmU7ywNud2gjwTjwo+LlzEo+B3eKD
-z8eeHcujbFBZciWd0R+PlGLKMr6gHoBP4nAvABEBAAG0I0FydW4gSSA8YXJ1bmlz
-YWFjQHN5c3RlbXJlYm9vdC5uZXQ+iQFXBBMBCABBAhsDBQsJCAcCBhUICQoLAgQW
-AgMBAh4BAheAAhkBFiEEf3MDQ/Lwnzx3v3nTLiXui2GAK7MFAmQAcAgFCRO2STAA
-CgkQLiXui2GAK7MWuAf7BOlCFYM9zUCBzzP8rNpE7554u6Uj0sivGb/WpXSa9nYe
-ofNmXbarsUyghPasGo3fhWc2Y4+WAkT3cx83xHm9pIe6zeBO6e6lb3g+qQv/gdTz
-u6arroEzc8GdzIP32T2mtbsZpI9nE7ggGNP2a6VsJN9f8pCqWI8Zv/R8HWVZvece
-CoLHGxOM2Nweg64PGJ0b6MfmCA1YJA7Pw946gamwitA92ANXdh6VyojSFIF6B1ue
-gO0p4UcdVI4vvJTUBRDLMLQMwn/oTxs+b340/oVd0L8oxniPQVjfpOaZiFluZnmW
-THzg1oEWcMKJE6peURdEIPP06JcBPhaDQa8MTCnU1IkBNwQTAQgAIQUCVi0wCgIb
-AwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgAAKCRAuJe6LYYArs/7nCACfiPKWI1h5
-roUjW1mfF3OXXm7P4un/wFNLC5mq2/zvrHmITWJZG6bPHJDohCex+TJUaC5imtbB
-ZV37OfDavgFXwjd+zcxxRA4Ozar8qutmqiOob8H6Sv3wDflLwmbqO1TxIS25rMcO
-JHnV/yeMOfx54khnYCmgTXtItE26wRETiwSNqLRqO9pE4regUUj91y0fa2O66GP7
-BPYFvusDclYwtQTP406m4aqLCv4O/Qn7n94WtBtIkRAl/ark198uZZa5IIfNiwrk
-9rSjXAQp7/5E25rQRUTZwK3UDHB9lXJj+7Jh5FcupQT9Lo11LIj2ouFDTf04I33e
-ZRhJaYzrD4mtiQFOBBMBCAA4AhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAFiEE
-f3MDQ/Lwnzx3v3nTLiXui2GAK7MFAljPXesACgkQLiXui2GAK7NHSggAyZuiP7bZ
-Yy0QZ95CbY3vUgZLpaJf9o8UfG+ER/uvPjcHD9wunPl/bCBjLoPf4pI2KgXwrLl5
-/mIbmx40CTRLBxW5CGgINHbDKRwfQH1XUWQ4EajoebkK2y1Mty31wmWE0/7POM77
-mWC/s6d1l09iBDYGoWTWEnL5a9PoOHmSzFXja5ZCuO7qqmjvxZOtc0dpb13GIv5g
-d1j8nGGcRJDnXku38lDvygd6dAqY1h0hihA1QgwC8RMCS80gaeGjf06+p5jeBofs
-zss7VgyhDgxte6kMRHJbBpK9oCjez7EhnPiU02+nIBQvp+StC+yMolI5Cjci7kgE
-/fHOijr5O6XClIkBUwQTAQgAPgIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgBYh
-BH9zA0Py8J88d7950y4l7othgCuzBQJasrFoBQkKaIqQAAoJEC4l7othgCuz9qgH
-+L4Bg8DMFRIDl7wLzB5yC8rZ3So9KW+qYlnUp1Ga1270ha+M8uPzjJ12xVke8CRz
-bxZeKuw+v6vbnMVxrae88A3vAiQmpRF+/jbCSTId7+ivzpo02LJsI7jH4J5pjQCI
-Lar70sWv2IjIJvjF4pwq+uHBw8J2wyxCyAEJ+BM6TFs87bPnpAcE8PfDgrb5uroU
-dHd8GTHWoYDdav+h5c3ABcJ6vsUBpRI1dDLvC4f5b32O24JSMlEm2OX6QS0KtbIE
-he4UEdhkaAkdKdfeRgfk9Tz71VoyeqOi20FkQ285JfI0bAHDIcxupzPzTD//EIbc
-js14C5yQh5oo5S5ehlPU5YkBVwQTAQgAQQIbAwULCQgHAgYVCAkKCwIEFgIDAQIe
-AQIXgAIZARYhBH9zA0Py8J88d7950y4l7othgCuzBQJecxFJBQkOKOpxAAoJEC4l
-7othgCuz5wAH/iz1f4ZUyqxQapFV1aEEUXewZV9uDYQJnhWHVM7VRe9r4I9QTm8b
-awrRcXCloJlqOoaZu2/MbGeOmhGvfqNLjzGsHnmXL8/rBzW03D4fO9uKcQh9kky0
-ZsM6xLLidfS12AQL/oxprpX1kLrAVUujoCB01b3Iy0weWMxNA00jz+e1L0/R9Ujw
-dNq+4zIFoZRp5k2hicOwgmtFa4AnbfD/KCxI2TSGsQB63XtXjZvhx4iqedZRNBwE
-RA/2/T0Qx/uNm9xlIeKmd5grgmxqtdi3vTKSiGIeLGQd8MaRlMo/P7h39ruoKmmF
-tkqxLevSN64rHuHEfrqGlbZJtFlWApgH5h20N0FydW4gSSAoTXkgcGVyc29uYWwg
-a2V5KSA8dGhlcm9hcm9mdGhlZHJhZ29uQGdtYWlsLmNvbT6JATEEMAEIABsFAlfl
-njwUHSBJJ20gbGVhdmluZyBnbWFpbC4ACgkQLiXui2GAK7M/pAgAgaji8T6JiEjk
-Cxg+xu42nFndwVeguk6f8U5ZlkLNM0XuYs/8EWT3eZM5LFVFjAYXsHK0MdAdQEYe
-fMbF5ZKzKtEXcuz7PAm5ZBu2q8fKo1uDjLyQDIxMRFBlSys6Ro3xx6/bZtYbl1H/
-D92tMIEIKxPRc1TEC2CKclu/35RmgAsctHHcEn5w8Nj+vSavlVsjy/41Zsn5MGoz
-oTXjmefpcunS2N8FmiUfliRcw8r33mxl0HlZgd5vGuYcjmN59PLsRM51h7Afaeye
-yNvoWoXLJy7JtgCM65fAXMVmXE0B1hLSxZVCaeUTAbx8n9BuBHA+cM+wcwAmzP0u
-6g0PX7nf24kBOQQTAQIAIwUCUitaWAIbAwcLCQgHAwIBBhUIAgkKCwQWAgMBAh4B
-AheAAAoJEC4l7othgCuzVHgH/jAyy7nfFDfEqJ97fxzDj9DdrSPTTEcm494iNq0V
-fiX6Wta9oIzBfGk7qpU3FkwRQnYzsj50XHnGYbYn3bZ254bdtTKC70N/fXATmWaS
-lIrTIBqlrqAmGV1gSCZmH+fMscgSvTNfKcBPNSTvnQwPdPqjrlUtPNqVMC45lqGQ
-LrLufZCagJ1yiWuigcHP93BRJ2RQws3YOtJ8tNPKhdiBtsVGYdn6ZaJh/oTj3eWy
-Ut8cRS2tHMLxqQgGs8BTIM5azAENDEKkQioD+BOWbMQ70ZdhMGkGq8G+RLg3haSL
-2y3eWawBZBI0+7x4dH/qXlwlmsn1uNKMAzRm1h79eQYbnZ6IXgQQEQgABgUCVQcB
-iQAKCRDc2ShkX3xJJkygAQCLtk5EWW8S/916lDhNi+R1OE8S6I/tTdjczqYTIa7E
-3wEA9L6RTECVsQ9J+duvLoukGIgqNHdRinoheBuIVAIHTIe5AQ0EUitaWAEIAJhZ
-VxJGITTgYG1EegfrfF8ZAOXu468VWljwmzgCArN/uMO9DliacrcQ3mjzocn+FrSo
-+Rf5zTOn2qIQ5zQoTuxfBxSGB6R3YowvBFQ58PYpfZoRScnkTuqKCxqvKH3AzyEW
-ZSuxZO6V4/OAn7QCICjQMCAlWOcvmfBsBIxGHutCAv5J3rax/pq2mwGx9LkvUPui
-FFkMidwsgxNLvBQyJIyoiN0n4spOGiJVn0LZz1C+tYEmwB7xtOtY6OdTO6q74Jyf
-anA3rspcXP+tQoRynwdCToHkU7gcJP1HQ8/7Entk9i6RbCITQwMriz9eML+V6zbo
-xYR0NmyjsZTAR+XTjl0AEQEAAYkBPAQYAQgAJgIbDBYhBH9zA0Py8J88d7950y4l
-7othgCuzBQJkAHAmBQkTtklOAAoJEC4l7othgCuzq2QIAMgq/g4Fv2kCLiEUwS46
-REO100Y7VRYZ1Gydiiv6TFJRhrZnXFvk+MqMg+ZbfXGKInJ7qrxT4g3/FPMHRivV
-U0+mCBwDVcsxgxx/54rwGvdg6CRd+1/NeAQc9bzS6/qaAnRc0cr8ChpDn8UwY0KS
-pBIVRQbNNXh13L9tV6SQSWCc90E/P1zwfxK27ItL6lJLo8xVLSe4mHpIC0Z1YZ4Z
-uCYgA6WP5SU+s6R22A+P5nroxB6Za6VD+kDstQ4bo/QsrLh3rSfJGD2+lQVWwH1/
-uvYzmXTFnVlSHOa0Zm0FQn0pj3goa3x1uPH8l3+JDu58/7DMU+/PvzkcSlZ13bwv
-Xb4=
-=fInu
+z8eeHcujbFBZciWd0R+PlGLKMr6gHoBP4nAvABEBAAHNN0FydW4gSSAoTXkgcGVy
+c29uYWwga2V5KSA8dGhlcm9hcm9mdGhlZHJhZ29uQGdtYWlsLmNvbT7CwHEEMAEI
+ABsFAlflnjwUHSBJJ20gbGVhdmluZyBnbWFpbC4ACgkQLiXui2GAK7M/pAgAgaji
+8T6JiEjkCxg+xu42nFndwVeguk6f8U5ZlkLNM0XuYs/8EWT3eZM5LFVFjAYXsHK0
+MdAdQEYefMbF5ZKzKtEXcuz7PAm5ZBu2q8fKo1uDjLyQDIxMRFBlSys6Ro3xx6/b
+ZtYbl1H/D92tMIEIKxPRc1TEC2CKclu/35RmgAsctHHcEn5w8Nj+vSavlVsjy/41
+Zsn5MGozoTXjmefpcunS2N8FmiUfliRcw8r33mxl0HlZgd5vGuYcjmN59PLsRM51
+h7AfaeyeyNvoWoXLJy7JtgCM65fAXMVmXE0B1hLSxZVCaeUTAbx8n9BuBHA+cM+w
+cwAmzP0u6g0PX7nf28LAeQQTAQIAIwUCUitaWAIbAwcLCQgHAwIBBhUIAgkKCwQW
+AgMBAh4BAheAAAoJEC4l7othgCuzVHgH/jAyy7nfFDfEqJ97fxzDj9DdrSPTTEcm
+494iNq0VfiX6Wta9oIzBfGk7qpU3FkwRQnYzsj50XHnGYbYn3bZ254bdtTKC70N/
+fXATmWaSlIrTIBqlrqAmGV1gSCZmH+fMscgSvTNfKcBPNSTvnQwPdPqjrlUtPNqV
+MC45lqGQLrLufZCagJ1yiWuigcHP93BRJ2RQws3YOtJ8tNPKhdiBtsVGYdn6ZaJh
+/oTj3eWyUt8cRS2tHMLxqQgGs8BTIM5azAENDEKkQioD+BOWbMQ70ZdhMGkGq8G+
+RLg3haSL2y3eWawBZBI0+7x4dH/qXlwlmsn1uNKMAzRm1h79eQYbnZ7CXgQQEQgA
+BgUCVQcBiQAKCRDc2ShkX3xJJkygAQCLtk5EWW8S/916lDhNi+R1OE8S6I/tTdjc
+zqYTIa7E3wEA9L6RTECVsQ9J+duvLoukGIgqNHdRinoheBuIVAIHTIfNI0FydW4g
+SSA8YXJ1bmlzYWFjQHN5c3RlbXJlYm9vdC5uZXQ+wsCXBBMBCABBAhsDBQsJCAcC
+BhUICQoLAgQWAgMBAh4BAheAAhkBFiEEf3MDQ/Lwnzx3v3nTLiXui2GAK7MFAmmk
+uZ0FCRlaksUACgkQLiXui2GAK7NMmwf/bMZPl3bXJc2RNKL0exHSu1mct5KqaCQW
+h5aq/K8YZyUPGOrmRxySJmzsGl5tir2fQH3eRMif9lAQcQHurGqhVR/WWVerO1uH
+XO7FDIZv95NTnxSE2UVTCUJQBbku9jCUv0D3/vRcoN2spJBMZWZ64Cu4n82mj4XO
+K3qn86WPQ5bHOVcJsVc+4yCGu7gRTln1hxsThUCOfuQcy9eRRg9hyqBpfu0vVnEp
+2ReCcxFpG/GVvllQSVO8DdsFrGpgEzlKy+MxGZAmniV+xDvpIN2CtMR13NFOslGF
+QZGgHERlQ9mG9aeuVIUZH+VlpEjGRnCIb8Usq40/ZS+V2ArCfowZaMLAlwQTAQgA
+QQIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgAIZARYhBH9zA0Py8J88d7950y4l
+7othgCuzBQJnw1tTBQkXeTR7AAoJEC4l7othgCuzGoEH/RBGwF1mC4Q+h5hqZTo/
+b7MeKCt+Jyr/9n9iVH05Iao0J+9+UUCZUd9w5ckCZcaHJnv9k7P25ToOagKhxlr0
+UwWisW8WIVlNLtkgyIzM+3xuQ4OUkMeADw53LmH/ZpP6rQFGbzJ2ecPYYWjdtXkP
+ds7yYyz4CZq2wVb+5SpBTHgPcv/tqqAfAY4WWM1Qebax0Y8nQgBPEzFNydItVhD9
+z2Y4o0BD8/rEa9sWcflpBNFt4jcEdk9BiyLHQry5lxg5wbWz8E7V1jbTT7KTfCHH
+sAkNT97CA8+pgPNpuoA6dRyXPbF8UoUqOa5x7nrN4qBcHBCRS3xXqhcCjkzo3xcw
+nCrCwJcEEwEIAEECGwMFCwkIBwIGFQgJCgsCBBYCAwECHgECF4ACGQEWIQR/cwND
+8vCfPHe/edMuJe6LYYArswUCZeGsFwUJFZeFPwAKCRAuJe6LYYArsy08B/9UDJ/V
+J/U4oUOugolenysKHSM6PL4Xxdg7pjIaYQgk9ZABk5peTVlqKBEX466N7OXFGuTK
+ajBJBKVlTCUocCyXZ9Wb/SdwNAZ6IcTMbn8l5IUDN7LNPuDmm1WIYg4Gyf2drYhq
+dxS/ufsC8wKQbKw1Xowp+OuLRVkoWggGBPCvmmtB22txoWzLFVoo7EYo8f47Q/+N
+cMDztbCpaWiKExgLXozfpXxS69WA0sf6BwQqsaLr3/5RcsPCqGz5eaSAbz17XHeF
+i5jNYpl4qcdd4oaYNNBUseC9LrhCPp9WWTTxqw+z4Jt3bZWVC02lx7wIVH6lCSG4
+p4DQki/uHL8Fx+OFwsCXBBMBCABBAhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheA
+AhkBFiEEf3MDQ/Lwnzx3v3nTLiXui2GAK7MFAmQAcAgFCRO2STAACgkQLiXui2GA
+K7MWuAf7BOlCFYM9zUCBzzP8rNpE7554u6Uj0sivGb/WpXSa9nYeofNmXbarsUyg
+hPasGo3fhWc2Y4+WAkT3cx83xHm9pIe6zeBO6e6lb3g+qQv/gdTzu6arroEzc8Gd
+zIP32T2mtbsZpI9nE7ggGNP2a6VsJN9f8pCqWI8Zv/R8HWVZveceCoLHGxOM2Nwe
+g64PGJ0b6MfmCA1YJA7Pw946gamwitA92ANXdh6VyojSFIF6B1uegO0p4UcdVI4v
+vJTUBRDLMLQMwn/oTxs+b340/oVd0L8oxniPQVjfpOaZiFluZnmWTHzg1oEWcMKJ
+E6peURdEIPP06JcBPhaDQa8MTCnU1MLAlwQTAQgAQQIbAwULCQgHAgYVCAkKCwIE
+FgIDAQIeAQIXgAIZARYhBH9zA0Py8J88d7950y4l7othgCuzBQJiLBhiBQkR4fGK
+AAoJEC4l7othgCuzYN4IAKlA0pe9t5JYDlGb/xxN/ndipfWcERbFPpuwUCVU9OJ5
+7stRJFqMe+rl8T7qNKA5XAylfQGDdrO4IwU+ua63sfwwpj0jEMamJGEdQvSSST1O
+gE1uQPrY4WSqjxKXns/IzlLOJvoocT7kC7cYZSAgUIAKKjYOTYLg+ouzEn2GLhe6
+CdGGhHYVi6s3Nkce+hNhj9glpAdJJYQD5eG11ocXjhsm8lvlz/ByufS2+CM9DSl4
+kqYtRkB7tZ+gz47R4+DIFLmh1e6KjSjqMvXzE7vuQDmzT+bafqo4AMgcv8mXCUAY
+srDwbKpX7a6AQwO80FyZKaEUVu+TtBCBnib8pOLL3KnCwJcEEwEIAEECGwMFCwkI
+BwIGFQgJCgsCBBYCAwECHgECF4ACGQEWIQR/cwND8vCfPHe/edMuJe6LYYArswUC
+YFTOBwUJEAqnLwAKCRAuJe6LYYArs91gB/9UXevvb4/Bquji/lC2gUtX1LINNIxV
+lftEL+5KBpkmNFNDMKyNK2QI8RRixvuA+/+FO9OrEPiLb+Bs49DKVb3ryVq4xICc
+R0BriNvrAvvJ92+XzpPGMB8aQy8xazdlkP6qwlwhlH9QuAB3jwc8bYPWkb955LaJ
+Onj9pCzX1KRKD+TiQR3ON2cub7XJNpsaQELJy50F/KfolIAbahfOYqZhQ7L1kqGK
+DCYp5h0yko6cC8MUdS3crH6QMEdeXaaTxAfZOsDpEeQspuR/CYgXEzwSsvyJaoZ7
+1ZvMFwcIE5bKQnnoY+R+fGynJwYq5yVvd24RkEMKsDTD9XImNkT2MxbLwsCXBBMB
+CABBAhsDBQsJCAcCBhUICQoLAgQWAgMBAh4BAheAAhkBFiEEf3MDQ/Lwnzx3v3nT
+LiXui2GAK7MFAl5zEUkFCQ4o6nEACgkQLiXui2GAK7PnAAf+LPV/hlTKrFBqkVXV
+oQRRd7BlX24NhAmeFYdUztVF72vgj1BObxtrCtFxcKWgmWo6hpm7b8xsZ46aEa9+
+o0uPMaweeZcvz+sHNbTcPh8724pxCH2STLRmwzrEsuJ19LXYBAv+jGmulfWQusBV
+S6OgIHTVvcjLTB5YzE0DTSPP57UvT9H1SPB02r7jMgWhlGnmTaGJw7CCa0VrgCdt
+8P8oLEjZNIaxAHrde1eNm+HHiKp51lE0HARED/b9PRDH+42b3GUh4qZ3mCuCbGq1
+2Le9MpKIYh4sZB3wxpGUyj8/uHf2u6gqaYW2SrEt69I3rise4cR+uoaVtkm0WVYC
+mAfmHcLAlwQTAQgAQQIbAwULCQgHAgYVCAkKCwIEFgIDAQIeAQIXgAIZARYhBH9z
+A0Py8J88d7950y4l7othgCuzBQJckMK2BQkMRpveAAoJEC4l7othgCuzR8gH/0vl
+UAiN9kpdXnfITyAdVangI300fR/1KT7XRHVCqtlIc3Q3kB/I7BP+tSJcJ6espkJB
+I4cXqzXNhfhrYvUUBxGSGDgo1E/K3YxS+DPndd6BIarWmBQGMXqavinbccOoQggG
+eRvyuN/+aWzTNZ/Yd9WSInOzzPvb1FrSHS8Mx7feTxaflkQN+2ZN+H+3D3hpU40e
+2ZzIUAcUALHhtuJScnhUvlWE9ezFRz9yQ8gfCPS5xTuaTa0XkG0kb0OHcNzpp4HN
+Pb3ErXfyIY/1o/RvO2NGWprCsXb0CW+d61kZLbI37xDKlYDOE20STO0TMoUbFDD5
+8veOlV9HJqwz7AJ/IvTCwJMEEwEIAD4CGwMFCwkIBwIGFQgJCgsCBBYCAwECHgEC
+F4AWIQR/cwND8vCfPHe/edMuJe6LYYArswUCWrKxaAUJCmiKkAAKCRAuJe6LYYAr
+s/aoB/i+AYPAzBUSA5e8C8wecgvK2d0qPSlvqmJZ1KdRmtdu9IWvjPLj84yddsVZ
+HvAkc28WXirsPr+r25zFca2nvPAN7wIkJqURfv42wkkyHe/or86aNNiybCO4x+Ce
+aY0AiC2q+9LFr9iIyCb4xeKcKvrhwcPCdsMsQsgBCfgTOkxbPO2z56QHBPD3w4K2
++bq6FHR3fBkx1qGA3Wr/oeXNwAXCer7FAaUSNXQy7wuH+W99jtuCUjJRJtjl+kEt
+CrWyBIXuFBHYZGgJHSnX3kYH5PU8+9VaMnqjottBZENvOSXyNGwBwyHMbqcz80w/
+/xCG3I7NeAuckIeaKOUuXoZT1OXCwI4EEwEIADgCGwMFCwkIBwIGFQgJCgsCBBYC
+AwECHgECF4AWIQR/cwND8vCfPHe/edMuJe6LYYArswUCWM9d6wAKCRAuJe6LYYAr
+s0dKCADJm6I/ttljLRBn3kJtje9SBkulol/2jxR8b4RH+68+NwcP3C6c+X9sIGMu
+g9/ikjYqBfCsuXn+YhubHjQJNEsHFbkIaAg0dsMpHB9AfVdRZDgRqOh5uQrbLUy3
+LfXCZYTT/s84zvuZYL+zp3WXT2IENgahZNYScvlr0+g4eZLMVeNrlkK47uqqaO/F
+k61zR2lvXcYi/mB3WPycYZxEkOdeS7fyUO/KB3p0CpjWHSGKEDVCDALxEwJLzSBp
+4aN/Tr6nmN4Gh+zOyztWDKEODG17qQxEclsGkr2gKN7PsSGc+JTTb6cgFC+n5K0L
+7IyiUjkKNyLuSAT98c6KOvk7pcKUwsCOBBMBCAAhBQJWLTAKAhsDBQsJCAcCBhUI
+CQoLAgQWAgMBAh4BAheAACEJEC4l7othgCuzFiEEf3MDQ/Lwnzx3v3nTLiXui2GA
+K7P+5wgAn4jyliNYea6FI1tZnxdzl15uz+Lp/8BTSwuZqtv876x5iE1iWRumzxyQ
+6IQnsfkyVGguYprWwWVd+znw2r4BV8I3fs3McUQODs2q/KrrZqojqG/B+kr98A35
+S8Jm6jtU8SEtuazHDiR51f8njDn8eeJIZ2ApoE17SLRNusERE4sEjai0ajvaROK3
+oFFI/dctH2tjuuhj+wT2Bb7rA3JWMLUEz+NOpuGqiwr+Dv0J+5/eFrQbSJEQJf2q
+5NffLmWWuSCHzYsK5Pa0o1wEKe/+RNua0EVE2cCt1AxwfZVyY/uyYeRXLqUE/S6N
+dSyI9qLhQ039OCN93mUYSWmM6w+Jrc7ATQRSK1pYAQgAmFlXEkYhNOBgbUR6B+t8
+XxkA5e7jrxVaWPCbOAICs3+4w70OWJpytxDeaPOhyf4WtKj5F/nNM6faohDnNChO
+7F8HFIYHpHdijC8EVDnw9il9mhFJyeRO6ooLGq8ofcDPIRZlK7Fk7pXj84CftAIg
+KNAwICVY5y+Z8GwEjEYe60IC/knetrH+mrabAbH0uS9Q+6IUWQyJ3CyDE0u8FDIk
+jKiI3Sfiyk4aIlWfQtnPUL61gSbAHvG061jo51M7qrvgnJ9qcDeuylxc/61ChHKf
+B0JOgeRTuBwk/UdDz/sSe2T2LpFsIhNDAyuLP14wv5XrNujFhHQ2bKOxlMBH5dOO
+XQARAQABwsB8BBgBCAAmAhsMFiEEf3MDQ/Lwnzx3v3nTLiXui2GAK7MFAmmkufYF
+CRlakx4ACgkQLiXui2GAK7OeLAgAn7CT3dvzgrenfBpqS6kgmonzCOFM6IqkBXW4
+3dWDpCX+Gn2WI7zDyGYPffMJaW1cN6gWpYerN2+nSWCLOiqM7SPMQ9jvPBshUaOc
+BkIzUdfGexk1YeUIfNuE/YhxUUpIG+3SEMzSkXEhLMWFsA5uqBfgPmM5x+k74TQO
+bAvcltQ515mUkitsOiDsD3IeMimjdWtF6K54vjq4mzlUAXfSTkEPGuLfLS3iJpSE
+dQoFywBCw776PeQT44VYCw0LRPsQUWv5SWdgJbO74wfLzSbOWdFdAeYeSQJx6eCq
+0W+w9aqcqw8NtaZimDPGD4TXqeXeB5IQ2rqDgveg6AyWJWAyTMLAfAQYAQgAJgIb
+DBYhBH9zA0Py8J88d7950y4l7othgCuzBQJnw1uFBQkXeTStAAoJEC4l7othgCuz
+pLkH/3HDDJ8u+MfXhKBgRaQTaYFZzQtPZrR1PcgTg9vz+UZ3c+oyMPz5pmFBWAIx
+hr8Z+31+XprnbQEYyRYALAsB5w79FDrUaDIhKQ4DhyWirgKdbQi9WkAew2Xfx6hj
+f8c/cK5SwROKqwwBoWXxrzbKvvmfTXny37ZG/g0is6Dk6MvrAwwsZRG/7xZpTZaI
+IRO4tO0uW9bd1DQwvLJeMW2ElDofTKYXwV267zm5yJpodr9Tssjj//eGZjkXuVXo
+5WhqzMTbJzuJpcKyekmtqzIocEcD5NIKhM4LYyjY8LeD3/tp4oy6VbAxtGox82BW
+DzP9T1SkGzJXCs1dqkT4Yv+SOWPCwHwEGAEIACYCGwwWIQR/cwND8vCfPHe/edMu
+Je6LYYArswUCZeGsLwUJFZeFVwAKCRAuJe6LYYArs4pZCACvhuwMVlRcdxR+MMLb
+zyzxElCun3VDbkDceIG8m0eGrProYO03j085ipPkzUFhlPJ8fP4W9HzhBCSWL7kO
+rIcWfZ0QEH0PkWcvPMRijiSefTfTtuUHUJc7oUDmbTtKUfS0xYOR1yO1zLfDk3kO
+qqvUrJP37OBtEEsXfp1BMOKTBUGJeLigRRoYnOZjVTR23HqEenP5iTkRyDaYYczh
+/UXbHAU14gDSyvD7mYou07ULWXneknmBiQZ6VGQHhVBf6zi7NVfl8NVMbKKBRQ9d
+4z0JCZO4xenR8FC5FklaEPsMLN/H4jc+HA4DEFygYcd6BICWhJLnRVMbSuLzWBp2
+zl37wsB8BBgBCAAmAhsMFiEEf3MDQ/Lwnzx3v3nTLiXui2GAK7MFAmQAcCYFCRO2
+SU4ACgkQLiXui2GAK7OrZAgAyCr+DgW/aQIuIRTBLjpEQ7XTRjtVFhnUbJ2KK/pM
+UlGGtmdcW+T4yoyD5lt9cYoicnuqvFPiDf8U8wdGK9VTT6YIHANVyzGDHH/nivAa
+92DoJF37X814BBz1vNLr+poCdFzRyvwKGkOfxTBjQpKkEhVFBs01eHXcv21XpJBJ
+YJz3QT8/XPB/Erbsi0vqUkujzFUtJ7iYekgLRnVhnhm4JiADpY/lJT6zpHbYD4/m
+eujEHplrpUP6QOy1Dhuj9CysuHetJ8kYPb6VBVbAfX+69jOZdMWdWVIc5rRmbQVC
+fSmPeChrfHW48fyXf4kO7nz/sMxT78+/ORxKVnXdvC9dvsLAfAQYAQgAJgIbDBYh
+BH9zA0Py8J88d7950y4l7othgCuzBQJiLBjYBQkR4fIAAAoJEC4l7othgCuzMPgH
+/jWB8bT+XLtTm55mUuRJYPEpu+yyvfsRCrzMTsoUM9eIcN7MP+cgX2UNaOgEnKPv
+V+LfmpJl7dydlVpC1xwE6viLpqy84UM7/5i4bEFbf8Me7uBxquwWtPhvMIk/SznM
+aQ5MGioW4J08dOWvR2J1+OXuXI0oJIuY9BZmiDItLE3zEaWLpn4Y61tdy64QoE+D
+bTu33btBHlhyxNC44GSSDC9UpqM2i4hH2RMeRdM502sJkHRo02q+Z/WXw+uUFE9V
+O5ZNvmTUemxSPABnla2hfv/QuINo363kKx8Fc8ws1RwYOOq5kgrAoFYK2vodpOH3
+EgDOyW34uWtWmmFaawEp45PCwHwEGAEIACYCGwwWIQR/cwND8vCfPHe/edMuJe6L
+YYArswUCYFTOSgUJEAqncgAKCRAuJe6LYYArs3uOB/43UO1QHPihvvgk+fxjnYR8
+SOYNwJeJnEzyzb7grWuAXC5BlVu39qyAYCPR8qGlukHmyNo8ansa0iEH5DaS/tWl
+NGrpCgq2MKQKKh0cDga6TLw9KkVI2y3acAHxz0R1nivppNz0NvuC59k2w2z1Wm1S
+hhlifF0eltwPoajWqxP2zsnX73sjKAwMORO5eXshmJ6f26ibBW8uGuKAyjgEGFo1
+3/7uBwNk0jloAemYAnKEHqh6QNq7vcFaM87vGvKpwoKd/EyoUQH4jNm4l/JLYS62
+G3NdscEW3QBKmf8G7Oes5ig3vT0wHIm9Db+MHWqJlZfIv3RvpBmptqMLSjRz5dQe
+wsB8BBgBCAAmAhsMFiEEf3MDQ/Lwnzx3v3nTLiXui2GAK7MFAl5zEXUFCQ4o6p0A
+CgkQLiXui2GAK7N+8Qf+JRu+S5tsE3zmvCW2xlIqyxOvvUzPFVh+TwSt1xhxEKq/
+WyowI/U8tdTVTAqR5/Ub+4cXoCeJLHk9XCsMBCQhi1OFl93BtB11qyQ5rrPxKX10
+5mCIXolpHwKIS6/tDKDC9qy0/dQqtYPlPYqOy7sl+HWMj035aUZfXBuYzZbtMkdY
+xpw1KmmMYdorDqqbqh1v2ERGrxbzKY3eZrXnBTO6u8zes4CEzRwEgJKPuktBXFZg
+f/iFk36NP69TfakcykjsRRoRYst0Yr1rC/zQ0QBQPlXFPjCQlpTu+LXtNH5bQpWQ
+nEsC0lvWj8q1DApU0zQ3K0+btgptOTNBx7P+txxG7sLAfAQYAQIAJgIbDBYhBH9z
+A0Py8J88d7950y4l7othgCuzBQJckMLMBQkMRpv0AAoJEC4l7othgCuzH3IH/jGz
+mXDhBwusuviCb10cg0f48e5BuUOgHE2HMR8gCt6naYdg63sWS3wWpSgmYPN2Soui
+YkV7i2qd+oP51tvYDjO7ymLU1szF6X3KlmeTwIQyTMi9XCnFXr3p0nJmYHC97lej
+GIb8iWX62dov3xUfkX27RTB0RxKXYMHp4kP9/mc1MtKJu/zs3sDoHcNhGpefoI10
+5mJY6+TXtXtlfTcNGgVkAqaPinhmizyf0tZ1VjxvTN2hb3sAN7ckdSfMnlBatSwC
+kZChbeI8zRcGnQbNxj0EvlFdJ54wusNHbYCVED9lB8y633uxYkpMWr5vqfoRhn+D
+16wesXtmuLOALmDSy0vCwHwEGAECACYCGwwWIQR/cwND8vCfPHe/edMuJe6LYYAr
+swUCWrKxiQUJCmiKsQAKCRAuJe6LYYArsw1zB/9kzmgVkyjNRKhNFZNQ8wa8ykD1
+XD8toCJapQ5Aa4Pu1Q2k+FzOt8GWK4/8C3AXAJOaE6frvtIXgBCQc5+owdUbxF1e
+Xm4piv2d72sQavV1OtNbjoL4mi86rz9i+FdZ9EDMQw8KFn5iDs0QwgP8BoofWQTl
+LaIHkx4Y8wPjSI56GhBlscm3/cFnIghAIRAQIOdpzwCN7DHJ1E4MTP3WIilgduBc
+VbE0HyBmpvy27feTUr6pH6nRYLslDwR39TEYc2vIYlXp0mJiV2RI98Ld7o3ZBTPq
+tVlNbdaqI8NOlnoNZOvpoWX5DuL8XUv3X2PymuiGesDPEPMacNnGCxm7HaKOwsB2
+BBgBAgAgAhsMFiEEf3MDQ/Lwnzx3v3nTLiXui2GAK7MFAljPXo4ACgkQLiXui2GA
+K7PtTwgAyO26xcPsKLAcO3MTzdujDAzF6x/RTNYujKnVr5BmgktgGwV0jIytIcWm
+/pqQBwDBmPqHYbM/KL8ke+HxyeBd1uR91BcAJMHcdaIqJg2zGPDL2xzsw78/68mL
+CF6Yn9zsFdjhWHv2n9UVqELxutmsk7dJF1uTdMTkSOjnnJ68MhaqAVGRKpc4paMg
+RNhHaFknKZ6Lu+GF8FajffUtLkkh9u0JlrFCbOfR3EbabJudXwUe0oVGWxnuKxXU
+4b+xjjZSjFYjAUejUKcx4x1Dia2oeOgpnIhSvkG66qn5fMb6BDLwA0f3FT75qR3d
++FLgyZBB8d9kA6JLAquU4hO9uzKza8LAZQQYAQIADwIbDAUCVu4g3wUJBqP6BwAK
+CRAuJe6LYYArs4nsCACNGwN6OPoYfRTO41jAJiniVaxHe3XPeuDjqzceczU8fWbl
+khVo/rLHuj9qM3sOR35Civ1LfolG2jixwWKLMW5T1NO80jo0yHDZLDFEpsW/wnC+
+3jxZFd2l+/BHQ/bBHOMXJ/zVOvezZX2PHuzJKSF1ngNzMW+hKFW8Hro7lBeb+DTm
+WDn3FhIZTbJhhcubkmZIF7JU1/oFDZ1YJVR4DNt/baYRAdC9ueIxpKDDpObkZHXp
+Z+nJfz7sg9HGm1TenUHAcuARSkesMGo8sn4sWxj4qiR/gHuMwoWZdGIAhBx9Did9
+/+e6QdnMc/JMOqOVfrPab8bb+kgCbTW7MJZz2rkCwsBlBBgBAgAPAhsMBQJVDLMB
+BQkEwowpAAoJEC4l7othgCuzexgH/1IZELhNl419bHI94wtQp+8ASTy5m6EnsbNM
+MYGEhXH2ClyBx1mgs8y0gPOzLRFgFgXckm1ySQc5gCw7dPBBq7YK7tDppSpgaSf3
+JCOqgUckgkCRFYIEJGa8AzYVgtT5uGhjEI6u8m2P5JQj76PrwrukvyVY5aI8nY/Y
+wUkdY+czywbzw0ZPImUKWP1mS2XceW3bM10PeO2cwipViEoY4/VKLM3IDrreIWVB
+fgobp63Jg+3UvK9Scl/fRCgnp3GmZabLiVWySvbwt8SOZsvc3DcfFwtwNhJp/BhC
+n2OsTUOviZkBUcZktf2sAWNG974ZaFhJjBSzNfV0scbsS6Pjq23CwF8EGAECAAkF
+AlIrWlgCGwwACgkQLiXui2GAK7Oc7gf/ZtYhV35VrABO9x5CFuuLeyxZc62KJ1gV
+cvaIZTme96XEQhFCmJsWRPLh0chhjOu5CNcNA+VNdYgF4g31gpUQHM/klDLBw8um
+AVM1td+hAfp8rR++9snI+DArISKKhoNyhKnhdpedyskJfY3X8EZoTL7c52ghWTe8
+5cDUIlvqS3WZts0NMyKk93ciJmpOTSDXTMp278/MSVOs7BVPHIko05dXrdRYy+I/
+LScERYaX/zHNiAYD++t40f1X0XPZRUCrQbrimxqxMu/yiw9pG9p7Ud/KbHxZt4pT
+wHuznK2KleIDV0KiYMhD7DkKoOn/ldr0CtHi5Eqins/xZCz/S2E03w==
+=hgaK
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/skribilo.spec b/skribilo.spec
index b9013eb..5ca8bb3 100644
--- a/skribilo.spec
+++ b/skribilo.spec
@@ -5,7 +5,7 @@
 
 Name:         skribilo
 Version:      0.11.0
-Release:      3%{?dist}
+Release:      4%{?dist}
 Summary:      A free document production tool
 
 License:      GPL-3.0-or-later
@@ -13,6 +13,10 @@ URL:          https://www.nongnu.org/%{name}
 Source0:      https://download.savannah.nongnu.org/releases/%{name}/%{name}-%{version}.tar.gz
 Source1:      https://download.savannah.nongnu.org/releases/%{name}/%{name}-%{version}.tar.gz.sig
 # https://systemreboot.net/about/arunisaac.pub
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - 7F730343F2F09F3C77BF79D32E25EE8B61802BB3 | sq keyring filter --experimental --cert 7F730343F2F09F3C77BF79D32E25EE8B61802BB3 --output update.pgp
+#   sq keyring merge --output arunisaac.pub.new arunisaac.pub update.pgp
+#   mv arunisaac.pub.new arunisaac.pub
 Source2:      arunisaac.pub
 
 BuildRequires:  emacs
@@ -29,6 +33,7 @@ BuildRequires:  texlive-collection-context
 BuildRequires:  glibc-common
 # needed to check signing
 BuildRequires:  gnupg2
+BuildRequires:  openpgpverify
 Requires:       guile30
 Requires:       lomt-junction-fonts
 Requires:       linux-libertine-biolinum-fonts
@@ -66,7 +71,7 @@ Configuration files for writing Skribilo documents using
 Emacs.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup
 
 iconv -f ISO-8859-1 -t UTF-8 ChangeLog.Skribe > ChangeLog.Skribe.tmp
@@ -195,6 +200,9 @@ make check
 %{_datadir}/emacs/site-lisp/skribe.el
 
 %changelog
+* Fri Oct 23 2026 Jakub Jelen <jjelen@redhat.com> - 0.11.0-4
+- Verify upstream source signatures with openpgpverify
+
 * Fri Jul 17 2026 Fedora Release Engineering <releng@fedoraproject.org> - 0.11.0-3
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_45_Mass_Rebuild
 
-- 
2.56.0

